packages/react/src/Testimonial/Testimonial.tsx

```diff
@@ -114,35 +115,7 @@ function _Name({children, className, position}: NameProps, ref) {
   )
 }
 
-export const Name = forwardRef(_Name)
```

Comment:
Heads up that there's a visual regression the Testimonial stories 👇.


Uploading Screen Recording 2023-02-22 at 13.54.02.mov…